src/Features/VisualBasic/Portable/Completion/CompletionProviders/InternalsVisibleToCompletionProvider.vb

```diff
@@ -0,0 +1,22 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.Completion.Providers
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Text
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
+
+    Friend NotInheritable Class InternalsVisibleToCompletionProvider
+        Inherits AbstractInternalsVisibleToCompletionProvider
+
+        Friend Overrides Function IsInsertionTrigger(text As SourceText, insertedCharacterPosition As Integer, options As OptionSet) As Boolean
+            Dim ch = text(insertedCharacterPosition)
+            Return ch = """"c
```

Comment:
This function is identical to the one for C#.  Do you need these overridden methods at all?